dev/puppetserver.conf

```diff
@@ -168,3 +168,12 @@ authorization: {
         }
     ]
 }
+
+certificate-authority: {
+   # This will house following new configuration settings 
+   # for infrastructure CRL
+   # Enable them to specify any non-default values
+   # - infra-crl-path:  
```

Comment:
would probably put `""` here to match up with the empty array you place after `compile-masters`

Reply:
Think the uncommented compile-masters accidentally got checked in. I can just comment them all out; the configuration parameter names are self explanatory on the expected values unless we want to provide some sample example values (while still leaving them commented).